### DISCUSSION TITLE
Extension Request for Campus Expert Registration Deadline

### DISCUSSION THREAD

**Discussion by Aadityachouhan1** [community/community#110537] [GitHub Education]
### Select Topic Area

Show & Tell

### Body

Dear Github,

I hope this message finds you well. I am writing to request an extension for the registration deadline for the Campus Expert program. As a dedicated advocate for [mention your organization or institution], I believe that extending the registration deadline will allow more passionate individuals to join and contribute to our community.

Given the high level of interest and the busy schedules of many potential participants, extending the deadline would provide an opportunity for those who may have missed the initial cutoff to apply and showcase their enthusiasm for becoming a Campus Expert. This extension would ensure a more inclusive and diverse cohort, enriching the program with a variety of perspectives and talents.

Moreover, extending the registration deadline aligns with our commitment to fostering a supportive and inclusive environment where every member has the chance to thrive and contribute meaningfully. By accommodating a broader range of applicants, we can strengthen our community and empower individuals to make a positive impact on campus and beyond.

I understand the importance of maintaining a structured timeline, and I assure you that I will work closely with the relevant teams to ensure a smooth transition if the extension is granted. Additionally, I am more than willing to assist with any additional tasks or responsibilities related to the extension process.

Thank you for considering this request. I am confident that extending the registration deadline will benefit our community and enhance the overall success of the Campus Expert program. Please feel free to reach out if you require any further information or clarification.

Looking forward to your favorable response.

Warm regards,

Aaditya Chouhan

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Aadityachouhan1 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

Please note: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by samus-aran** [comment]
Hi @Aadityachouhan1 

Thank you for your interest in applying for GitHub Campus Experts. The application period has now come to an end. The next in take is [August 2024](https://education.github.com/campus_experts). Good Luck 👍
